fleet_vehicle_inspection/models/fleet_vehicle_inspection_line.py

```diff
@@ -56,7 +54,7 @@ class FleetVehicleInspectionLine(models.Model):
     )
 
     inspection_line_image_ids = fields.One2many(
-        "fleet.vehicle.inspection.line.image", "inspection_line_id", string="Images"
+        "fleet.vehicle.inspection.line.image", "inspection_line_id"
```

Comment:
Not sure removing this field's `string` makes sense.
The auto generated string is too long.